packages/scenario/test/choice-name.test.ts

```diff
@@ -38,10 +39,13 @@ describe('JavaRosa ports: ChoiceNameTest.java', () => {
 	});
 
 	describe('choice name call in repeat with static choices', () => {
-		it.fails('gets choice name', async () => {
+		it('gets choice name', async () => {
 			const scenario = await Scenario.init(r('jr-choice-name.xml'));
+
 			scenario.answer('/jr-choice-name/my-repeat[1]/select_one', 'choice4');
+			scenario.createNewRepeat('/jr-choice-name/my-repeat');
```

Comment:
I needed to add this to get the tests to pass, but I think it's correct. Before this it was trying to update the second repeat item but there was only one child.